src/util/color/color.cpp

```diff
@@ -0,0 +1,53 @@
+#include "util/color/color.h"
+
+#include "util/math.h"
+
+namespace Color {
+
+const PredefinedColorsSet kDefaultPredefinedColorsSet = PredefinedColorsSet();
```

Comment:
kDefaultPredefinedColorsSet reads double almost like a "Weißer Schimmel"
There can only be one color set predefined, so we should remove "default". 
Actually even "predefined" is superfluid, because it is not possible to have an other color set, but that's an other story.

Reply:
If we remove the "Default" from the name then the whole constant becomes pointless, because it is just the a default constructed PredefinedColorsSet. I just tried to get the code into a somehow consistent state. I'm not willing to discuss design decision that have been made by someone else.